.github/workflows/win.yml

```diff
@@ -0,0 +1,44 @@
+name: windows
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        CONDA_PY: ["36", "37"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        auto-activate-base: true
+        activate-environment: ''
+    - name: Install
+      env:
+        CONDA_PY: ${{ matrix.CONDA_PY }}
+      run: |
+        conda config --system --set always_yes yes --set changeps1 no
+        conda config --system --append channels conda-forge
+        conda config --system --append channels esss
+        conda install -c conda-forge conda-devenv
+        conda info -a
+        conda devenv
+    - name: Tests
+      run: |
+        conda activate esss-fix-format-py${{ matrix.CONDA_PY }}
+        conda info -a
```

Comment:
I added this for debug purposes only. I will remove as soon as we have Win build fixed.

Reply:
The value displayed for **PATH** env var makes it look like the env vars set by conda devenv are not taking hold. Maybe this is a problem with a new conda on windows.

You could try to disable `auto-update-conda` and force a old version of conda (4.8.3 comes to mind). If it works we could narrow down the latest version where it works... If is a bug we will need to fix it on conda-devenv or conda itself.

Reply:
Hmmmm! Great, @prusse-martin! Thanks a lot, I will test some ways to use an older version of conda.

Reply:
Pinned to `4.8.3`, no luck so far.

Reply:
Strange... try adding this after `conda activate`:

```
echo PYTHONPATH=%PYTHONPATH%
where python
```